homeassistant/components/image_processing/__init__.py

```diff
@@ -43,6 +43,11 @@
 ATTR_FACES = "faces"
 ATTR_GENDER = "gender"
 ATTR_GLASSES = "glasses"
+ATTR_HAIR = "hair"
```

Comment:
I agree, these should not be added to default integration. Add to microsoft integration instead.